libraries/chain/db_maint.cpp

```diff
@@ -951,7 +949,6 @@ void database::process_bitassets()
 void process_hf_1465( database& db )
 {
    const auto head_num = db.head_block_num();
-   wlog( "Processing hard fork core-1465 at block ${n}", ("n",head_num) );
```

Comment:
Note: this function changed nothing in mainnet, so perhaps remove it, but perhaps need it for testnet.

Reply:
Yes. Needed for testnet. Added comment.